src/main/java/meditracker/medication/Medication.java

```diff
@@ -129,6 +129,7 @@ public void setRepeat(String repeat) {
 
     @Override
     public String toString() {
-        return getName() + " | " + getQuantity() + " | " + getDosage();
+        return getName() + " | " + getQuantity() + " | " + getDosage() + " | " +
+                getExpiryDate() + " | " + getIntakeFreq() + " | " + getRemarks();
```

Comment:
Please do not include getDosage and getIntakeFreq. We will be removing them soon.

Reply:
Copy that. Working on it right away.

Reply:
Resolved in [543ee25](https://github.com/AY2324S2-CS2113T-T09-1/tp/pull/88/commits/543ee25c3506852d62de3ee5073138f83ccea26c). However needed more commits such as [4deb0e6](https://github.com/AY2324S2-CS2113T-T09-1/tp/pull/88/commits/4deb0e6d086ac0b942b7b6c9cceaac0289fd8a4f) and [ad83c8c](https://github.com/AY2324S2-CS2113T-T09-1/tp/pull/88/commits/ad83c8c2b245a9079aaee70be9f5e6992b110ac3) due to the changes done.